src/utils.ts

```diff
@@ -35,3 +37,20 @@ export function buildDomainWideDelegationJWT(
 
   return JSON.stringify(body);
 }
+
+/**
```

Comment:
I'll probably move this into actions-utils, but I wanted something that was slightly faster for iteration right now.